update-versions.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Prepare new artman release.
+
+This script updates SHA hashes of latest commits for both gapic-generator
+and googleapis, and updates version number for subsequent (auto-)release.
+
+usage: update-versions.py [-h] [--fetch-googleapis] [--fetch-gapic-generator]
+                          [version]
+"""
+
+import argparse
+import os.path
+import re
+import requests
+import sys
+from ruamel.yaml import YAML
+
+dockerfile_googleapis_regex = r'^(ENV GOOGLEAPIS_HASH )\S+()'
+dockerfile_gapic_generator_regex = r'^(ENV GAPIC_GENERATOR_HASH )\S+()'
+dockerfile_version_regex = r'^(ENV ARTMAN_VERSION )\S+()'
+setup_py_version_regex = r"^(current_version = ')\S+(')"
+
+github_gapic_generator_project = 'googleapis/gapic-generator'
+github_googleapis_project = 'googleapis/googleapis'
+
+
+def get_latest_commit_sha(project):
+    url = 'https://api.github.com/repos/%s/branches/master' % project
+    # try to get GitHub credentials from config
+    auth = None
+    artman_config = os.path.join(os.environ['HOME'], '.artman', 'config.yaml')
+    try:
+        yaml = YAML()
+        with open(artman_config) as config_yaml:
+            config = yaml.load(config_yaml)
+            github_username = config['github']['username']
+            github_token = config['github']['token']
+            auth = requests.auth.HTTPBasicAuth(github_username, github_token)
+    except:
+        print('Cannot get GitHub token from %s, trying anonymously...' %
```

Comment:
Of note: people might generally not have this set because github functionality in artman is broken (and it will even probably be removed). Maybe we should print a message describing how the config could be added?

Also, this failure doesn't distinguish between being unable to open the artman config and hitting an error accessing github.